slippyMap/resources/geoJsonOptions.js

```diff
@@ -18,5 +18,16 @@ export default {
   },
   style: feature => {
     return feature.properties;
+  },
+  coordsToLatLng: coords => {
+    const lat = coords[1];
+    let lng = coords[0];
+    const pacificBounds = Leaflet.latLngBounds([[-180, -90], [-125, 90]]);
```

Comment:
could you add a comment here explaining why this is doing what it's doing. will be helpful for our future selfs.